gulpfile.js

```diff
@@ -0,0 +1,33 @@
+
+// used by travis-ci to check the hiera json files
+
+var gulp = require("gulp");
+var jsonlint = require("gulp-jsonlint");
+var expect = require('gulp-expect-file');
+var gutil = require('gulp-util');
```

Comment:
Within the OAE project we have the styleguide of ordering the require statements based on the variable name. This idiom combined with a sane naming strategy makes it easy to quickly scan the require statements and see what dependencies are being used by a module.

So this should become:
```javascript
var expect = require('gulp-expect-file');
var gulp = require("gulp");
var gutil = require('gulp-util');
var jsonlint = require("gulp-jsonlint");
```